crates/ide-assists/src/handlers/extract_variable.rs

```diff
@@ -135,6 +135,7 @@ pub(crate) fn extract_variable(acc: &mut Assists, ctx: &AssistContext<'_>) -> Op
                             }
                         }
                     }
+                    edit.rename();
```

Comment:
nit: Can pull this out of the match block to after the match block

Reply:
Done